methods/transient/NISAR_SES_Transient_Requirement_Validation.ipynb

```diff
@@ -6,99 +6,50 @@
     "tags": []
```

Comment:
<p>Please add another cell with some text like this to highlight this final result for Approach 1:</p><pre class="ql-syntax" spellcheck="false">&lt;div class="alert alert-warning"&gt;
Final result Method 2—
&nbsp; &nbsp; 68% of points below the requirements line is success
&lt;/div&gt;


</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/nisar-solid/ATBD/pull/29/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='94'/>

Reply:
@kanglcn could you address this suggestion too?